tower-balance/Cargo.toml

```diff
@@ -27,7 +27,9 @@ futures = "0.1.26"
 indexmap = "1.0.2"
 log = "0.4.1"
 rand = "0.6.5"
+tokio-sync = "0.1.3"
 tokio-timer = "0.2.4"
+tower = { version = "0.1.0", path = "../tower" }
```

Comment:
Doesn't this result in a circular dependency?

Reply:
Cargo culted from tower-buffer, tower-test, & tower-util...  I'm happy to do whatever is the best practice; though, for the time being, `tower` does not export tower-balance...

Reply:
`tower` can be in the `dev-dependencies` section safely, but it probably should not depend directly on `tower`.

Reply:
In general, I want to update the dev-dep on "facade" crates to use a `*` version dependency. Without that, there will be conflicts when we try to issue breaking change releases...